packages/manager/apps/hpc-vmware-managed-vcd/mocks/vcd-organization/vcd-organization.handler.ts

```diff
@@ -1,28 +1,61 @@
 import { PathParams } from 'msw';
 import { Handler } from '../../../../../../playwright-helpers';
 import { organizationList } from './vcd-organization.mock';
+import { datacentreList } from './vcd-datacentre.mock';
 
 export type GetOrganizationMocksParams = {
   isOrganizationKo?: boolean;
+  isOrganizationUpdateKo?: boolean;
   nbOrganization?: number;
   allOrgsBackedUp?: boolean;
+  isDatacentresKo?: boolean;
+  nbDatacentres?: number;
 };
 
 const findOrganizationById = (params: PathParams) =>
   organizationList.find(({ id }) => id === params.id);
 
 export const getOrganizationMocks = ({
   isOrganizationKo,
+  isOrganizationUpdateKo,
   nbOrganization = Number.POSITIVE_INFINITY,
   allOrgsBackedUp,
+  isDatacentresKo,
+  nbDatacentres = Number.POSITIVE_INFINITY,
 }: GetOrganizationMocksParams): Handler[] => {
   const nb = allOrgsBackedUp ? 1 : nbOrganization;
   return [
+    {
+      url: '/vmwareCloudDirector/organization/:id/virtualDataCenter',
```

Comment:
you can't use some route constants here?

Reply:
I will fix them in next TU PR